src/Sulu/Component/Webspace/Analyzer/Attributes/PortalInformationRequestProcessor.php

```diff
@@ -55,6 +55,10 @@ public function process(Request $request, RequestAttributes $requestAttributes)
         }
 
         $attributes['localization'] = $portalInformation->getLocalization();
+        if ($portalInformation->getLocalization()) {
```

Comment:
I would use `$attributes['localization']`, because we can save another function call that way.